scripts/init.zsh

```diff
@@ -58,6 +58,10 @@ zstyle -T :znap: auto-compile &&
     add-zsh-hook preexec ..znap.auto-compile
 add-zsh-hook zsh_directory_name ..znap.dirname
 
+if zstyle -s ':znap:pull' auto-update _ ; then
```

Comment:
I'm not sure whether it's better to test here or not. We have to test in the function, or pass the interval as a parameter (which just means setting a local here in init). My thinking was that we can avoid reading the file if the style is not set 🤷